### PR TITLE
Disable access logs

### PR DESCRIPTION
**Pull request by tamas-molnar**
* refactored app to enable disabling of access log
* existing apps should not be affected by the change
* do we still need the "local" case? I'd remove it to simplify things.

### CODE REVIEW

File: baseftrwapp/baseftapp.go [hunk 1]
```diff
@@ -17,6 +17,15 @@ import (
 	metrics "github.com/rcrowley/go-metrics"
 )
 
+type RWConf struct {
+	Engs          map[string]Service
```
Comment (martingartonft) [inline baseftrwapp/baseftapp.go]: Can we rename Engs to Services (which I think is an overdue change)?

File: baseftrwapp/baseftapp.go [hunk 2]
```diff
@@ -26,39 +35,48 @@ import (
...
 		if err != nil {
 			log.Fatalf("Service for path %s could not startup, err=%s", path, err)
 		}
 	}
 
-	//	http.Handle("/", m)
-
-	if env != "local" {
-		f, err := os.OpenFile("/var/log/apps/"+serviceName+"-go-app.log", os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+	//TODO do we still need this?
```
Comment (tosan88) [inline baseftrwapp/baseftapp.go]: I think this was added for FTP2 services, which still use it?

Comment (martingartonft) [inline baseftrwapp/baseftapp.go]: I think keeping compatibility for now is sensible. In future we can deprecate/ignore that param if needed.

**Comment by tosan88** [comment]
:+1:

**Comment by martingartonft** [comment]
+1

**Comment by catalin-ilea** [comment]
+1

**Pull request merged**